test-eslint.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
-npm install -g eslint
-npm install -g eslint-config-angular
-npm install -g eslint-plugin-angular
-npm install -g eslint-config-dustinspecker
+npm install -g eslint eslint-config-angular eslint-plugin-angular eslint-config-dustinspecker
```

Comment:
Note that these won't actually be included unless specified by the project in an `.eslintrc`.